src/App/Fossa/Analyze.hs

```diff
@@ -290,6 +290,8 @@ uploadSuccessfulAnalysis (BaseDir basedir) apiOpts metadata override units = do
   -- Warn on contributor errors, never fail
   void . Diag.recover . runExecIO $ tryUploadContributors basedir apiOpts (uploadLocator uploadResult)
 
+  logStdout . decodeUtf8 . Aeson.encode $ buildProjectSummary revision (uploadLocator uploadResult) buildUrl
```

Comment:
Should we do this before trying contributors?  I don't know if it makes a difference.